openmetadata-ui/src/main/resources/ui/src/components/common/OwnerWidget/OwnerWidget.tsx

```diff
@@ -160,7 +162,11 @@ const OwnerWidget = ({
           return false;
         }
 
-        return userPermissions[Operation.EditOwner];
+        return hasPemission(
+          Operation.EditOwner,
+          entityType as EntityType,
```

Comment:
Instead use as to type cast can we make sure that type of `entityType` will always be `EntityType`